node/lib/util/merge_util.js

```diff
@@ -0,0 +1,621 @@
+/*
+ * Copyright (c) 2016, Two Sigma Open Source
```

Comment:
2017

Reply:
unless this is just copied from earlier, I guess

Reply:
I'll update it anyway.